src/HomePanel.ts

```diff
@@ -130,20 +130,20 @@ export class HomePanel {
     // );
 
     // Uri to load styles into webview
-    const stylesResetUri = webview.asWebviewUri(
-      vscode.Uri.joinPath(
-        this._extensionUri,
-        "media",
-        "reset.css"
-      )
-    );
-    const stylesMainUri = webview.asWebviewUri(
-      vscode.Uri.joinPath(
-        this._extensionUri,
-        "media",
-        "vscode.css"
-      )
-    );
+    const styleResetUri = webview.asWebviewUri(
+			vscode.Uri.joinPath(this._extensionUri, "media", "reset.css")
+		);
+		const styleVSCodeUri = webview.asWebviewUri(
+			vscode.Uri.joinPath(this._extensionUri, "media", "vscode.css")
+		);
+
+    const scriptUri = webview.asWebviewUri(
+			vscode.Uri.joinPath(this._extensionUri, "out", "compiled/home.js")
+		);
+		const styleMainUri = webview.asWebviewUri(
+			vscode.Uri.joinPath(this._extensionUri, "out", "compiled/home.css")
```

Comment:
```suggestion
			vscode.Uri.joinPath(this._extensionUri, "out", "compiled/Home.js")
		);
		const styleMainUri = webview.asWebviewUri(
			vscode.Uri.joinPath(this._extensionUri, "out", "compiled/Home.css")
```

Check the "compiled" folder, I had a "Home.js" instead of "home.js" file. When I changed this it seemed to hook up nicely!

Reply:
Hi @aitikgupta how's it going?  I hope things are going excellently and you are not still languishing in our extension code.  I just read your comment.  I also have a Home.js file, and that's after I got this "(!) Plugin svelte: 'ext_vscode' is not defined" error.  

What does that mean?  Does it mean that Home.svelte -> Home.ts is compiling into Home.js when it really should be compiling into home.js?  @ShrillShrestha how are you getting lowercase home.js?

Reply:
When using Rollup and TypeScript concurrently to compile Home.svelte, what does it mean when macOS capitalizes Home.js while other operating systems such as Windows might not?  Why do we have to Command + R to get the extension running again after npm run watch?  What are the applications of acquireVsCodeApi?  Anyway, this is such a great PR @ShrillShrestha I've been reviewing it so much because I wanted to merge it.  But don't merge this PR!  We are definitely using svelte now.